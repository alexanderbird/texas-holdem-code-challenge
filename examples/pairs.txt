2H 4C 6S 8D TC
Anderea 6C JC
Zane 6D 3C
Donni 8C JS
Beaufort 8S 3D
