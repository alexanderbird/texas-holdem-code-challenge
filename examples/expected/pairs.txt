1 Donni Pair 8 (kicker Jack)
2 Beaufort Pair 8 (kicker 10)
3 Anderea Pair 6 (kicker Jack)
4 Zane Pair 6 (kicker 10)
